packages/apps-config/src/endpoints/testingRococo.ts

```diff
@@ -73,7 +73,7 @@ export function createRococo (t: TFunction): LinkOption[] {
       },
       {
         info: 'rococoBifrost',
-        paraId: 107,
+        paraId: 10,
```

Comment:
Any id < 1000 is now reserved for system parachains. Not sure if this is correct.

Reply:
Confirmed with Shawn, it needs to be > 1000

Reply:
now update to 1024